### DISCUSSION TITLE
NginxProxyManager LetsEncryptPlugins not working

### DISCUSSION THREAD

**Discussion by nicedevil007** [tteck/Proxmox#1411] [Help Needed]
### A clear and concise description of the issue.

Hey tteck,

I wasn't sure if this is a new problem on the LXC with latest 2.10.2 or if this existed earlier but I'm not able to use the netcup plugin to get my wildcard certificate with your LXC container. I was hoping it will be fixed with NginxProxyManager 2.10.3 but it is not :(

I run the old fashioned way now to implement a docker container with default nginx setup and did the exact same entries for the plugin... there it worked like a charm on first try.

### Please provide detailed steps to reproduce the issue.

If I hit OK on the first run of nginx proxy to get my wildcard it first do the waiting procedure screen for about 30 sec - 60 sec and then it throws me back to the configure screen of the plugin with a red bar at the top without any issue / fault inside of it.

On the working installation with docker it was quite simple a 600 sec (yes 10 minutes) waiting to get the propagation of the dns settings on netcup. But then it just works.

**Comment by tteck** [comment]
I'm not sure this is a script issue, or a missing feature. Did you try running the command `pip install certbot-dns-netcup`?

**Comment by nicedevil007** [comment]
> 

I'm not sure as well but with this error here (yes did your pip command before) I can't tell you what is going on with the LXC :/

![image](https://github.com/tteck/Proxmox/assets/17103076/aedf7449-2881-448a-b0c2-9f1ee44df4e0)

**Comment by nicedevil007** [comment]
trying with complete new LXC container, fresh install of version 2.10.3 now

**Reply by nicedevil007** [reply]
again... now it tells me Bad Gateway after 15 minutes....
docker works, script not

**Reply by tteck** [reply]
Is Netcup DNS a commonly used service? As I mentioned earlier, I moved from Nginx Proxy Manager to Cloudflared and am satisfied with my decision. It seems that Netcup doesn't provide a similar service.

**Reply by nicedevil007** [reply]
It is a well known Webhoster/DNS Provider/Datacenter in germany. To move everything to a provider not in my country isn't a good idea because of DNS pings etc f.e.

I don't know if the other DNS challenge plugins from any other provider are working for a wildcard f.e.

But it is ok to run NGINX with a docker again if that is one thing that can't be solved inside an LXC

**Comment by Kelsch** [comment]
> Is Netcup DNS a commonly used service? As I mentioned earlier, I moved from Nginx Proxy Manager to Cloudflared and am satisfied with my decision. It seems that Netcup doesn't provide a similar service.

How did you get Cloudflare to work? I just keep getting error 522. Which I know it's saying that my host is down or that my isp change my ip, but I checked that and everything else I was able to google.

I've tried Proxied, DNS only, Full, Full (strict), Flexible, Off (not secure), Always Use HTTPS, and every combination of those and I still can't get anything but error 522
